nixos/modules/services/security/vaultwarden/default.nix

```diff
@@ -1,11 +1,27 @@
 { config, lib, pkgs, ... }:
 
-with lib;
-
 let
   cfg = config.services.vaultwarden;
   user = config.users.users.vaultwarden.name;
   group = config.users.groups.vaultwarden.name;
+  inherit (lib)
```

Comment:
I would suggest that we drop this in this PR and I take care myself to remove with lib. Agree?

#295016